spec/lib/daemons/rails/configuration_spec.rb

```diff
@@ -8,23 +8,13 @@
 
   describe "Default configuration" do
     describe "rails env" do
-      its(:root) { should == Rails.root }
-      its(:daemons_path) { should == Rails.root.join('lib', 'daemons') }
+      it { expect(subject.root).to eq Rails.root }
+      it { expect(subject.daemons_path).to eq Rails.root.join('lib', 'daemons') }
     end
 
     describe "no rails" do
-      before :all do
-        Dir.chdir Rails.root
-        Object.const_set :Rails_, Rails
-        Object.send :remove_const, :Rails
-      end
-      after :all do
-        Object.const_set :Rails, Rails_
-        Object.send :remove_const, :Rails_
-        Dir.chdir Rails.root.parent.parent
-      end
-      its(:root) { should == Rails_.root }
-      its(:daemons_path) { should == Rails_.root.join('lib', 'daemons') }
+      it { expect(subject.root).to eq Rails.root }
+      it { expect(subject.daemons_path).to eq Rails.root.join('lib', 'daemons') }
```

Comment:
This test intended to check if code still work without Rails lib, but after rewriting it is same as "rails env"

Reply:
Hmm...i think you're right. I will add more tests in that direction.